libsolidity/codegen/ir/IRGenerationContext.cpp

```diff
@@ -111,10 +112,12 @@ string IRGenerationContext::internalDispatch(size_t _in, size_t _out)
 		vector<map<string, string>> functions;
 		for (auto const& contract: mostDerivedContract().annotation().linearizedBaseContracts)
 			for (FunctionDefinition const* function: contract->definedFunctions())
+			{
+				FunctionType const* functionType = TypeProvider::function(*function)->asCallableFunction(false);
 				if (
 					!function->isConstructor() &&
```

Comment:
```suggestion
				if (
					FunctionType const* functionType = TypeProvider::function(*function)->asCallableFunction(false);
					!function->isConstructor() &&
```
That way you can get rid of the brackets in the for :-). But not important.